cmd/hunt.go

```diff
@@ -95,6 +95,7 @@ func init() {
 	huntCmd.MarkFlagRequired("file")
 
 	huntCmd.Flags().BoolVarP(&commandOutput, "include-cmd-output", "c", false, "Include the executed command output")
+	huntCmd.Flags().BoolVarP(&commandError, "include-cmd-error", "e", false, "Include the executed command error")
```

Comment:
```suggestion
	huntCmd.Flags().BoolVarP(&commandError, "include-cmd-stderr", "e", false, "Include the executed command error")
```


```suggestion
	huntCmd.Flags().BoolVarP(&commandError, "include-cmd-errors", "e", false, "Include the executed command error")
```

Reply:
We might change also the `include-cmd-output` to `include-cmd-stdout` to be consistent

Reply:
I committed the wrong option 😅
Could you add the commit suggestion again?

Reply:
You can simply edit locally